context/app/static/js/components/Providers.jsx

```diff
@@ -18,13 +18,18 @@ const AppContext = React.createContext({});
 
 function Providers(props) {
   const { endpoints, groupsToken, isAuthenticated, userEmail, children, workspacesToken } = props;
+  // TODO: Delete this when workspaces are publicly released.
+  // If we stay in limbo for a long time, this configuration could be moved out of code.
+  const workspacesUsers = ['chuck_mccallum@hms.harvard.edu'];
```

Comment:
'pdblood@andrew.cmu.edu', 'jpuerto@andrew.cmu.edu'

Reply:
'dbetancur@cmu.edu'

This is fine for the initial list, we are working on getting a few more users together, but this does not have to wait for that list to be complete. Once I hear back w/ RE to that, I will let the team know.